ixdiagnose/plugins/audit.py

```diff
@@ -35,12 +35,19 @@ class Audit(Plugin):
         CommandMetric(
             # This generates a file that is collected in the associated FileMetric.
             'truenas_verify_data', [
+                Command(
+                    ['truenas_verify'], 'Result from truenas_verify', serializable=False,
+                    safe_returncodes=[],  # With no safe_returncodes we can silently run the command
+                ),
                 Command(
                     ['shasum', '-a', '256', '/conf/rootfs.mtree'], 'sha256 rootfs.mtree', serializable=False
                 ),
                 Command(
-                    ['truenas_verify'], 'Result from truenas_verify', serializable=False,
-                    safe_returncodes=[],  # With no safe_returncodes we can silently run the command
+                    ['aureport'], 'auditd summary report', serializable=False
+                ),
+                Command(
+                    ['ausearch --format csv -m ALL --start week-ago --end now | head -101 | grep -v TTY'],
```

Comment:
I was requesting that you query our system audit table like middleware above. Not do ausearch.